modules/web/src/app/core/services/node-data/service.ts

```diff
@@ -140,6 +141,18 @@ export class NodeDataService {
     } as NodeData;
   }
 
+  getDefaultOS(): OperatingSystem {
+    if (this.isProvider(NodeProvider.ANEXIA)) {
+      return OperatingSystem.Flatcar;
+    }
+    return OperatingSystem.Ubuntu;
```

Comment:
What if the provider doesn't support Ubuntu? Instead of assuming and hardcoding I'd recommend that we should use `isOperatingSystemSupported` to check if it supports ubuntu and only in that case we should return `Ubuntu`. Something that we were already doing in the original method 

https://github.com/kubermatic/dashboard/blob/main/modules/web/src/app/node-data/component.ts#L434